test/dynamo/test_export.py

```diff
@@ -2411,6 +2411,29 @@ def f(x, y):
         buffer = io.BytesIO()
         torch.save(gm, buffer)
 
+        def test_export_with_inline_constrains(self):
+            def f(x):
+                a = x.item()
+                constrain_as_size(a, 4, 7)
+                return torch.empty((a, 4))
```

Comment:
Could you add another test where we actually use `torch.empty((a, 4))` somewhere non-trivially? Just wanted to make sure the symbols are propagating properly.